perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -295,21 +297,94 @@ public Group call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+
+	/*#
+	 * Adds members to a group. If already a member of the group, the member will be skipped.
+	 * Non-empty list of members expected, if empty, no member will be added.
+	 *
+	 * @param groups List<Group> List of groups that the member should be added to <code>id</code>
```

Comment:
This docs seem to be a copy of ``addMember()`` including params, which are wrong for ``addMembers()``. Param ``groups`` should be a ``group`` and ``member`` should be ``members`` in this case